Source/WebKit/WebProcess/WebPage/DrawingArea.h

```diff
@@ -203,6 +203,7 @@ class DrawingArea : public RefCounted<DrawingArea>, public IPC::MessageReceiver,
     const DrawingAreaType m_type;
     DrawingAreaIdentifier m_identifier;
     WeakRef<WebPage> m_webPage;
+    Ref<WebPage> protectedWebPage() const;
```

Comment:
Nit: As a general principle, we separate functions from variables rather than interleaving them. List all the functions first and then the variables. For example, I'd put `protectedWebPage` below `send`.

The same idea applies to `protectedConnection`, `protectedRemoteLayerTreeContext`, and `protectedCommitQueue`.